app/views/categories/show.html.erb

```diff
@@ -0,0 +1,50 @@
+<section class="categories_controllers">
+  <header class="input-container">
+    <nav>
+      <ul class='links'>
+        <li>
+        <%= link_to categories_path do %>
+          <i class="fa fa-arrow-left"></i>
+          <% end %>
+        </li>
+        <li>
+          DETAILS
+        </li>
+        <li>
+          <i class="fa fa-bars"></i>
+        </li>
+      </ul>
+    </nav>
+  </header>
+ <section class="category-lists-wrapper details ">
+    <div class='desc'>
```

Comment:
[OPTIONAL] Please make sure only the owner users of the category can access and see their categories. I created another session with another user and I can still access the categories that were created by another user.